### PR TITLE
switch schema refs that use tag to uris

### PR DESCRIPTION
**Pull request by braingram**
asdf will soon warn for refs that are not uris. This PR changes the few remaining `$ref`s that contain tags to uris.

The tag `tag:stsci.edu:asdf/transform/transform-1.1.0` resolves to `http://stsci.edu/schemas/asdf/transform/transform-1.1.0`. All uses of this tag were replaced with the corresponding uri.

Resolves [JP-nnnn](https://jira.stsci.edu/browse/JP-nnnn)
Resolves [RCAL-nnnn](https://jira.stsci.edu/browse/RCAL-nnnn)

Closes #

This PR addresses ...

**Checklist**
- [x] added entry in `CHANGES.rst` (either in `Bug Fixes` or `Changes to API`)
- [ ] updated relevant tests
- [ ] updated relevant documentation
- [ ] updated relevant milestone(s)
- [x] added relevant label(s)

### CODE REVIEW

**Comment by braingram** [comment]
Regression tests running at: https://plwishmaster.stsci.edu:8081/job/RT/job/JWST-Developers-Pull-Requests/899/

**Review by hbushouse — APPROVED** [review]
Looks OK to me. This has no effect on existing jwst code, right?

**Comment by braingram** [comment]
> Looks OK to me. This has no effect on existing jwst code, right?

Correct!

tldr; asdf internally converts `tag:stsci.edu:asdf/transform/transform-1.1.0` to `http://stsci.edu/schemas/asdf/transform/transform-1.1.0` due to a known "quirk" which will be fixed in asdf 3.0

It's a bit convoluted but old asdf extensions allowed defining a `tag_mapping` (which allowed converting tag string like `tag:stsci.edu:asdf/transform/transform-1.1.0` to url strings like `http://stsci.edu/schemas/asdf/transform/transform-1.1.0`). Due to an internal "quirk" of asdf this tag mapping is applied when a schema is loaded (see: https://github.com/asdf-format/asdf/issues/752) which allowed tag strings to be used as jsonschema `$ref`s. The refs updated here all rely on the `tag_mapping` provided by the asdf `BuiltinExtension` (which will be removed in asdf 3.0). asdf 3.0 will also start warning when a tag is used for a `$ref`.
